examples/seismic/tutorials/16_variable_z_topography.ipynb

```diff
@@ -0,0 +1,549 @@
+{
```

Comment:
<p>Should explain why this is the case. Also, may be worth referencing the subdomains notebook here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1900/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>